SonarQube.MSBuild.Tasks/WriteProjectInfoFile.cs

```diff
@@ -31,18 +32,17 @@ public class WriteProjectInfoFile : Task
         [Required]
         public string FullProjectPath { get; set; }
 
-        [Required]
+        /// <summary>
+        /// Optional, in case we are imported into a project type that does not have a language specified
+        /// </summary>
         public string ProjectLanguage { get; set; }
 
         public string ProjectGuid { get; set; }
 
-        [Required]
         public bool IsTest { get; set; }
 
-        [Required]
         public bool IsExcluded { get; set; }
 
-        [Required]
```

Comment:
Null appeared to be a valid value in any case for arrays so removing this attribute doesn't seem to make any difference.